docs/ism/refresh-analyzer.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Refresh Search Analyzer
+parent: Index State Management
+nav_order: 6
+---
+
+# Refresh search analyzer
+
+With ISM installed, you can refresh search analyzers in real time with the following API:
+
+```json
+POST /_opendistro/_refresh_search_analyzers/<index or alias or wildcard>
+```
+For example, if you change the synonym list in your analyzer, the change takes effect without you needing to close and reopen the index.
```

Comment:
Please note that the refresh_search_analyzer API only refreshes search time synonym analyzers which are defined with `"updatable": true` flag like below

```
"analysis" : {
                "analyzer" : {
                    "my_synonyms" : {
                        "tokenizer" : "whitespace",
                        "filter" : ["synonym"]
                    }
                },
                "filter" : {
                    "synonym" : {
                        "type" : "synonym_graph",
                        "synonyms_path" : "synonyms.txt",
                        "updateable" : true
                    }
                }
            }

```

Reply:
@ashwinkumar12345